### PR TITLE
ORT Web build script

### PR DESCRIPTION
**Pull request by shalvamist**
**Description**: Describe your changes.
Adding a few scripts to enable user to build ORT Web in a more direct way
This simplifies the build & deployment process for the user

### CODE REVIEW

**Review by fs-eire — CHANGES_REQUESTED** [review]
please fix the pylint warnings.

File: js/scripts/build_web.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            + " --build_dir "
+            + os.path.join('"' + BINARIES_DIR, entry["dir"] + '"')
+            + " "
+            + entry["wasm_flags"]
+        )
+        p = subprocess.Popen(command, shell=True)
+        p.wait()
+        if(not os.path.exists(os.path.join(BINARIES_DIR, entry["dir"], configuration, entry["wasm_file_name"]))and (p.returncode!=0)):
+            print("Error - can find " + entry["wasm_file_name"] + " there might be an issue with the build\n")
+            exit()
```
Comment (fs-eire) [inline js/scripts/build_web.py:86]: use non-zero exit code

Comment (shalvamist) [reply js/scripts/build_web.py:86]: I think this code covers non-zero exit code - "(p.returncode!=0)"
Can you please clarify ?

Comment (fs-eire) [reply js/scripts/build_web.py:86]: no, I mean the `exit()` call ( which is suggested by pylint to use `sys.exit()` ) should use a non-zero parameter. https://docs.python.org/3/library/sys.html#sys.exit

**Review by fs-eire — CHANGES_REQUESTED** [review]
please revert changes to submodules. those are not a part of the PR.

**Review by fs-eire — APPROVED** [review]

**Review by fs-eire — APPROVED** [review]

**Pull request merged**